server/api/user/user.controller.js

```diff
@@ -0,0 +1,30 @@
+const controller = {};
+
+controller.getUserPoolExpenses = (req, res) => {
+  res.send(`getUserPoolExpenses ${req.params.id}, ${req.params['exp-pool-id']}`);
+};
+
+controller.getUserPools = (req, res) => {
+  res.send(`getUserPools for user id: ${req.params.id}`);
+};
+
+controller.getUserExpenses = (req, res) => {
+  res.send(`getUserExpenses for user id: ${req.params.id}`);
+};
+
+controller.getUser = (req, res) => {
+  res.send(`getUser for user id: ${req.params.id}`);
+};
+
+controller.addUserExpense = (req, res) => {
+  res.send('addUserExpense');
+};
+
+controller.addUserToPool = (req, res) => {
+  res.send('addUserToPool');
+};
+
+controller.addUser = (req, res) => {
+  res.send('addUser');
+};
```

Comment:
Let's have a space after this line to be consistent with the other controller file.